test/support/rails_application_fixture_helper.rb

```diff
@@ -28,36 +18,28 @@ def use_template(template)
       raise "Unknown fixture template #{template}"
     end
 
-    RailsPaths.root(app_dir)
+    root = Pathname.new(app_dir)
+    Rails.application.config.stubs(:root).returns(root)
   end
 
   private
 
   def set_load_paths_for_minimal_template
-    RailsPaths.autoload(to_app_paths("/components/sales/app/models"))
-    RailsPaths.eager_load([])
-    RailsPaths.autoload_once([])
+    main = Zeitwerk::Loader.new
```

Comment:
I overlooked the fact that this is introducing a direct dependency on Zeitwerk here until I saw [these tests failing](https://github.com/Shopify/packwerk/runs/5792482153?check_suite_focus=true). Given that Packwerk depends on Zeitwerk's structural assumptions (and indeed is even named after it?) I don't think adding an explicit dependency on Zeitwerk would be that much of a stretch. However, I'm tempted to just use a mock object since we only need the one `:root_dirs` method here anyway.

Do others have thoughts on the best direction here?